www/shared/components/cookie-banner/CookieBanner.module.css

```diff
@@ -0,0 +1,14 @@
+@import "../../styles/imports/variables";
+
+.cookieBanner {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    height: 8rem;
+    padding: 0 2.5rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-top: 1px solid;
+    background-color: color(var(--color-white));
```

Comment:
Is it necessary to use the color function if you're not doing any calculations? 🤔